packages/components/src/Disclosure/src/InternalDisclosureContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from "react";
```

Comment:
mentionned earlier, but i don't think this is required. you can most likely use useSlot instead